java/java.editor/nbproject/project.properties

```diff
@@ -17,7 +17,7 @@
 
 javadoc.title=Java Editor
 
-spec.version.base=2.78.0
+spec.version.base=2.79.0
```

Comment:
What is the reason behind increasing the spec version? Are we changing something public in API/SPI?

Reply:
Thanks @lkishalmi for the review. I'm not sure if this change is actually an API change, and if so, where this API change should be documented (there are no apichanges.xml and arch.xml files in the java/java.editor, all changes in the production code apply only to private methods)? Should I add the since tag to private methods or not increment the specification version at all?